src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -1,23 +1,25 @@
-import { withKnobs, boolean } from "@storybook/addon-knobs";
 import Dropdown from ".";
 
 export default {
-  title: "Dropdown",
-  decorators: [withKnobs],
+  title: "Components/Dropdown",
   component: Dropdown,
+  args: {
+    disabled: false,
+  },
 };
 
-export const Story = () => (
-  <Dropdown disabled={boolean("disabled", false)} buttonText="Test">
-    Some Children
-  </Dropdown>
-);
+export const Story = (args) => <Dropdown {...args}>Some Children</Dropdown>;
+Story.args = {
+  disabled: false,
+  text: "Test",
+};
```

Comment:
Do you need this and `CustomButtonRender.args` on `L23`? 🤔 I removed them and it seems like it still works the same way, with the disable functionality